sources/packages/api/src-sql/EducationProgramsOfferings/Add-tuition-remittance.sql

```diff
@@ -0,0 +1,15 @@
+-- Add tuition_remittance_requested_amount
```

Comment:
Please would be good to start the file name with "Rollback" to be easy to follow.